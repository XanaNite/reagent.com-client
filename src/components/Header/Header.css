.headerSection h1{
    padding-left: 10px;
}

.headerSection nav {
    padding-left: 10px;
    padding-bottom: 2px;
    background-color: rgb(37, 23, 71);
    color: seashell;
    text-align: right;
}

.headerSection p{
    font-size: 11px;
    margin: 3px;
}

.headerSection span{
    padding-left: 20px;
}

.headerSection nav a{
    color: seashell;
    text-decoration: none;
    margin-right: 5px;
}

@media screen and (min-width: 760px){
    .headerSection p{
        font-size: 15px;
        margin: 5px;
    }
}