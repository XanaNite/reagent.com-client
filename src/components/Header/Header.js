import React from 'react'
import {Link} from 'react-router-dom'
import './Header.css'

export default function Header(){
    return(
    <header role="banner" className='headerSection'>
        <nav role="navigation">
            Agent Dashboard: <span><Link to="/login">Log In</Link></span> <span><Link to="/register">Sign Up</Link></span>
        </nav>
        <h1><Link to="/">REagent.com</Link></h1>
    </header>
    )
}