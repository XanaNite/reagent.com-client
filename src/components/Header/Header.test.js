import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './Header';

describe(`Header component`, () => {
    const props = {
        className: 'test-class-name',
        children: <p>test children</p>,
        'data-other': 'test-other-prop',
        agent: {
            first_name: "test",
            last_name: "test",
            title: "test",
            city: "test",
            state: "test"
        }
    };

    it('renders the Header given props', () => {
        const wrapper = shallow(<Header {...props} />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
})