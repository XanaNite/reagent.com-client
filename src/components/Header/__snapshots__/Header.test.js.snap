// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the Header given props 1`] = `
<header
  className="headerSection"
  role="banner"
>
  <nav
    role="navigation"
  >
    Agent Dashboard: 
    <span>
      <Link
        to="/login"
      >
        Log In
      </Link>
    </span>
     
    <span>
      <Link
        to="/register"
      >
        Sign Up
      </Link>
    </span>
  </nav>
  <h1>
    <Link
      to="/"
    >
      REagent.com
    </Link>
  </h1>
</header>
`;
